<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>480</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Click Mapper</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Window {
	background-color: #111111;
	border-radius: 20px;
}

#MacroListCtr {
	border-radius: 40px;
	background-color: #0A0A0A;
}
#MacroList {
	background: transparent;
	border-bottom-left-radius: 20px;
	border-bottom-right-radius: 20px;
}
#MacroList QWidget {
	background: transparent;
}

#MacroListTitle {
background-color: #1C1C1C;
	border-top-left-radius: 20px;
	border-top-right-radius: 20px;
}

#MacroListTitleLabel {
	font: 30pt &quot;Inter&quot;;
	color: white;
}




QScrollBar:vertical{
	width: 20px;
	right: 20px;
	margin: 10px 10px 45px 0;
	background: #151515;

}
QScrollBar::handle:vertical {
	background: #505050;
	border-radius: 5px;
}
QScrollBar::handle:vertical:hover {
	background: #5A5A5A;
}
QScrollBar::handle:vertical:pressed {
	background: #6A6A6A;
}
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical
{
      border: none;
      background: none;
      color: none;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
	background: #151515;
}

#MacroListContent &gt; QFrame { 
	border-radius: 10px;
	min-height: 65px;
	max-height: 65px; 
}
#MacroListContent QFrame:hover {
	background: #151515; 
}
#MacroList QLabel { 
	color: white;
	font: 12pt &quot;Inter&quot; 
}
#MacroList QPushButton { 
	border-image: url(:/TitleBar/title_bar/ClsBtn.svg);
	min-width: 20px;
	max-width: 20px;
	min-height: 20px;
	max-height: 20px; 
}
#MacroList QPushButton:hover { 
	border-image: url(:/TitleBar/title_bar/ClsBtn_Hov.svg); 
}
#MacroList QPushButton:pressed { 
	border-image: url(:/TitleBar/title_bar/ClsBtn_Act.svg); 
} 

#MacroListFilterInput {
	font: 10pt &quot;Inter&quot;;
	background: #333333;
	border-radius: 10px;
	color: #cccccc;
	padding: 5px 10px
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="BarPlaceholder">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="Main">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QFrame" name="MacroListCtr">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="MacroListTitle">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="MacroListTitleLabel">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>60</height>
               </size>
              </property>
              <property name="text">
               <string>Saved Macros</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="MacroListFilterInput">
              <property name="placeholderText">
               <string>Filter...</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="MacroList">
           <property name="cursor" stdset="0">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="MacroListContent">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>350</width>
              <height>20</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>30</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/resources.qrc"/>
 </resources>
 <connections/>
</ui>
